Projects/App/Sources/Presentation/MainHome/MainHomeViewController.swift

```diff
@@ -0,0 +1,317 @@
+//
+//  MainHomeViewController.swift
+//  App
+//
+//  Created by kokojong on 2023/02/12.
+//  Copyright © 2023 com.promise8. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import RxCocoa
+import RxSwift
+
+final class MainHomeViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let bag = DisposeBag()
+    var viewModel: MainHomeViewModel?
+    enum Section: CaseIterable {
+        case proceeding
+        case ended
+    }
+    var isOneStepPaging = true
+    var currentIndex: CGFloat = 0 {
+        didSet {
+            pageLabel.customAttributedString(text: "\(Int(currentIndex)+1)/5", highlightText: "\(Int(currentIndex)+1)", textFont: .www(size: 14)!, highlightTextFont: .www(size: 14)!, textColor: .wwwColor(.Gray450), highlightTextColor: .wwwColor(.WWWGreen))
+        }
+    }
```

Comment:
p3) 너무 길어서 쪼개는건 어떨까용?
```suggestion
    var currentIndex: CGFloat = 0 {
        didSet {
            pageLabel.customAttributedString(text: "\(Int(currentIndex)+1)/5",
                                             highlightText: "\(Int(currentIndex)+1)",
                                             textFont: .www(size: 14)!,
                                             highlightTextFont: .www(size: 14)!,
                                             textColor: .wwwColor(.Gray450),
                                             highlightTextColor: .wwwColor(.WWWGreen))
        }
    }
```